packages/jellyfish-buffer/__tests__/composer.test.ts

```diff
@@ -254,6 +254,75 @@ describe('ComposableBuffer.varUIntArray', () => {
   })
 })
 
+describe('ComposableBuffer.varUIntArrayRaw', () => {
```

Comment:
There is another alternative if you wanna consider it. It's creating a new C Composable called `CPoolPairId`.

Reply:
That design encapsulates the buffer writing/reading layer, hence much easier. This design exposes

Reply:
Not sure if you're referring to same structure, what I had tried previously was `CPoolPairId extend ComposableBuffer<number>)` which resulted data = [{ }, { }, ...] instead of [number, number, ...]

Reply:
I like @fuxingloh idea for the same reasons stated. Should be doable IMO. We have similar work done already🤔
But can refactor later since this is a blocking issue.